### PR TITLE
fix  the path to bluetooth_mapper

### PR DESCRIPTION
**Pull request by jedyang**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: docs/mappers/bluetooth_mapper.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Note: Before trying to push the docker image to the remote repository please ens
 ### Deploying bluetooth mapper application
 
 ```shell
-cd $GOPATH/src/github.com/kubeedge/kubeedge/device/bluetooth_mapper
+cd $GOPATH/src/github.com/kubeedge/kubeedge/mapper/bluetooth_mapper
```
Comment (fisherxu) [inline docs/mappers/bluetooth_mapper.md]: it's `mappers` now :) Thanks!

**Comment by jedyang** [comment]
em.. mappers

**Comment by fisherxu** [comment]
Why close it :) We can modify then force push @jedyang

**Comment by fisherxu** [comment]
open another pr is also fine :)

**Pull request closed without merging**